tests/test_cases/input/no_solidus_escape.kdl

```diff
@@ -0,0 +1 @@
+node "\\"
```

Comment:
Shouldn't there also be an output case for this, and one for backslash then?

Seeing as the escaped form was removed